info/token-info-storage-enq.json

```diff
@@ -1,107 +1,21 @@
-            "linkedin":"https://www.linkedin.com/some-token",
-            "youtube":"https://www.youtube.com/some-token",
-            "qq":"https://qm.qq.com/some-token",
-            "weixin":"https://weixin.qq.com/some-token",
-            "telegram":"https://t.me/some-token"
-        }, 
-        "description": {
-            "en":"EN text about some token",
-            "de":"DE text about some token",
-            "fr":"FR text about some token",
-            "ru":"RU text about some token",
-            "pt":"PT text about some token",
-            "vi":"VI text about some token",
-            "in":"IN text about some token",
-            "tr":"TR text about some token"
-        },
-        "coingecko_id": "coingecko-id"             
-    },
-    {
-        "token_id":"another-token-id",
-        "website":"https://anotherTokenWebsite.com",
-        "media": {                            
-            "facebook":"https://www.facebook.com/another-token",
-            "twitter":"https://twitter.com/another-token",
-            "medium":"https://medium.com/another-token",
-            "github":"https://github.com/another-token",
-            "forklog":"https://hub.forklog.com/another-token",
-            "bitcointalk":"https://bitcointalk.org/another-token",
-            "linkedin":"https://www.linkedin.com/another-token",
-            "youtube":"https://www.youtube.com/another-token",
-            "qq":"https://qm.qq.com/another-token",
-            "weixin":"https://weixin.qq.com/another-token",
-            "telegram":"https://t.me/another-token"
-        },
-        "description": {
-            "en":"EN text about another token",
-            "de":"DE text about another token"
-        },
-        "coingecko_id": "coingecko-id"             
+            "en": "SURC tokens are imitated by the First Volga Region Exchange of Digital Financial Assets to collect primary investments. Further, during the IPO on the Moscow Stock Exchange, tokens will be redeemed taking into account interest or convected into shares.",
+            "de": "Die SURC-Token werden von der Ersten Wolga-Börse für digitale Finanzanlagen simuliert, um Primärinvestitionen zu sammeln. Als nächstes werden die Token bei einem Börsengang an der Moskauer Börse unter Berücksichtigung von Zinsen zurückgekauft oder in Aktien umgewandelt.",
+            "fr": "Les jetons SURC sont simulés Par la première bourse d'Actifs Financiers Numériques de la Volga pour la collecte des investissements primaires. Ensuite, lors de l'introduction en bourse à la bourse de Moscou, les jetons seront rachetés en tenant compte des intérêts ou convertis en actions.",
+            "ru": "Токены SURC имитированы Первой Поволжской Биржей Цифровых Финансовых Активов для сбора первичных инвестиций. Далее при IPO на Московской Бирже токены будут выкуплены с учетом процентов или конвектированы в акции.",
+         },
+        "coingecko_id": "SURC"             
```

Comment:
Such coingecko_id does not exist. Please remove this field (or add a valid coingecko_id).